docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.html

```diff
@@ -10,7 +10,7 @@
             [allowNull]="allowNull"
             [maxHeight]="maxHeight"
             [placeholder]="placeholder"
-            aria-label="Filter input">
+            [aria-label]="'Filter input'">
```

Comment:
```suggestion
            aria-label="Filter input">
```
You can use the following technique to remove the attribute from the host wrapper:
https://stackblitz.com/edit/angular-ivy-luwfpc?file=src%2Fapp%2Fhello.component.ts,src%2Fapp%2Fapp.component.html